src/App.tsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string | null>(null);
```

Comment:
you can store just an empty string in your input value like initial value
```suggestion
  const [query, setQuery] = useState('');
```

Reply:
Done